<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addbooks</class>
 <widget class="QMainWindow" name="addbooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>419</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Books</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <property name="spacing">
         <number>15</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:15px;</string>
            </property>
            <property name="text">
             <string>Publisher</string>
            </property>
            <property name="buddy">
             <cstring>bookPublisher</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_10">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:15px;</string>
            </property>
            <property name="text">
             <string>Source</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="bookTitle">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">outline:none;
border:0.5px solid #333;
padding:7px 15px;
font-size:15px;</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:15px;</string>
            </property>
            <property name="text">
             <string>Pages</string>
            </property>
            <property name="buddy">
             <cstring>pages</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:15px;</string>
            </property>
            <property name="text">
             <string>Title</string>
            </property>
            <property name="buddy">
             <cstring>bookTitle</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="bookPublisher">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">outline:none;
border:0.5px solid #333;
padding:7px 15px;
font-size:15px;</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="pages">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">outline:none;
border:0.5px solid #333;
padding:7px 15px;
font-size:15px;</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="source">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">outline:none;
border:0.5px solid #333;
padding:7px 15px;
font-size:15px;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:15px;</string>
            </property>
            <property name="text">
             <string>Author</string>
            </property>
            <property name="buddy">
             <cstring>bookAuthor</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="copies">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:15px;
padding:5px 15px;</string>
            </property>
            <item>
             <property name="text">
              <string>Single</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Multiple</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_9">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:15px;</string>
            </property>
            <property name="text">
             <string>Copies</string>
            </property>
            <property name="buddy">
             <cstring>copies</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="bookAuthor">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">outline:none;
border:0.5px solid #333;
padding:7px 15px;
font-size:15px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>12</number>
          </property>
          <property name="rightMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <property name="spacing">
           <number>15</number>
          </property>
          <item row="4" column="0">
           <widget class="QLabel" name="label_8">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:15px;</string>
            </property>
            <property name="text">
             <string>Publication Year</string>
            </property>
            <property name="buddy">
             <cstring>publishedYear</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="endingLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:15px;</string>
            </property>
            <property name="text">
             <string>Ending No</string>
            </property>
            <property name="buddy">
             <cstring>endNo</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="endNo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">outline:none;
border:0.5px solid #333;
padding:7px 15px;
font-size:15px;</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="startingLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:15px;</string>
            </property>
            <property name="text">
             <string>Starting No</string>
            </property>
            <property name="buddy">
             <cstring>startNo</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDateEdit" name="publishedYear">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">outline:none;
border:0.5px solid #333;
padding:7px 15px;
font-size:15px;</string>
            </property>
            <property name="displayFormat">
             <string>yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="startNo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">outline:none;
border:0.5px solid #333;
padding:7px 15px;
font-size:15px;</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_7">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:15px;</string>
            </property>
            <property name="text">
             <string>Price</string>
            </property>
            <property name="buddy">
             <cstring>price</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="price">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">outline:none;
border:0.5px solid #333;
padding:7px 15px;
font-size:15px;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:15px;</string>
            </property>
            <property name="text">
             <string>Edition</string>
            </property>
            <property name="buddy">
             <cstring>place</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="place">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">outline:none;
border:0.5px solid #333;
padding:7px 15px;
font-size:15px;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:15px;</string>
            </property>
            <property name="text">
             <string>Publication Place</string>
            </property>
            <property name="buddy">
             <cstring>Edition</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="Edition">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">outline:none;
border:0.5px solid #333;
padding:7px 15px;
font-size:15px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="closeBtn">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:0 10px;
font-size:15px;</string>
          </property>
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearBtn">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:0 10px;
font-size:15px;</string>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addBtn">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:0 10px;
font-size:15px;</string>
          </property>
          <property name="text">
           <string>Add to Registry</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>bookTitle</tabstop>
  <tabstop>bookAuthor</tabstop>
  <tabstop>bookPublisher</tabstop>
  <tabstop>pages</tabstop>
  <tabstop>Edition</tabstop>
  <tabstop>place</tabstop>
  <tabstop>publishedYear</tabstop>
  <tabstop>price</tabstop>
  <tabstop>startNo</tabstop>
  <tabstop>endNo</tabstop>
  <tabstop>addBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
